uninstall.sh

```diff
@@ -396,6 +396,15 @@ uninstall() {
     # Uninstall all cli
     uninstall_cli
 
+    if using_hooks_path; then
```

Comment:
Moved that up, here in favor of:
1. clean dependant config stuff
2. clean all githooks config variables...